app/forms/state_file/id_disability_form.rb

```diff
@@ -28,12 +41,12 @@ def save
     private
 
     def clean_up_followups
-      primary_followups =  @intake.filer_1099_rs(:primary).map(&:state_specific_followup).compact
       if primary_disabled == "no" || %w[spouse none].include?(mfj_disability)
+        primary_followups =  @intake.filer_1099_rs(:primary).map(&:state_specific_followup).compact
```

Comment:
🤔 this is more for my own understanding, but why do you delete the followups?

Reply:
this is due to:
https://github.com/codeforamerica/vita-min/blob/main/app/lib/efile/id/id39_r_calculator.rb#L87-L95

If the user goes through the flow first for the 62-65 year old filer, answer "yes" for disabilty, then answer "eligible_income_source_yes?" for that eligible 1099R, but then comes back via the review page & then answers "no" for disabled. Without the clean up, we will add up all the income for 1099Rs. 

I suppose another way to handle it is to:
1. keep the followup (whatever the answer was -- eligible/not eligible)
2. check if the filer is associated with a particular filer who is either disabled/over 65(senior) in the calculation

Reply:
quick followup here https://github.com/codeforamerica/vita-min/pull/5665 while my mind is on it but no need to get it through review or anything, unless we feel like we have loads of time to run through this quickly before release.